plugins/catalog-react/src/components/UserListPicker/UserListPicker.tsx

```diff
@@ -111,13 +114,26 @@ function getFilterGroups(orgName: string | undefined): ButtonGroup[] {
       items: [
         {
           id: 'all',
-          label: 'All',
+          label: t('all'),
         },
       ],
     },
   ];
 }
 
+const locals = {
```

Comment:
**suggestion:** Maybe we can move this to a separate file to give an example of how load translation in the plugin config load. WDYT?

Reply:
Yeah, this just demo. Probably in most scenarios, we do not need to add locals in plugin. Just need to defined which namespace which key this plugin will consume, add set fallback value to `t()`

```
const { t } = useTranslation('catalog');

<span>t('owned', 'Owned')</span>
```